src/project-manager.ts

```diff
@@ -357,7 +357,7 @@ export class ProjectManager implements Disposable {
 				const info = ts.preProcessFile(contents, true, true);
 				const compilerOpt = config.getHost().getCompilationSettings();
 				for (const imp of info.importedFiles) {
-					const resolved = ts.resolveModuleName(imp.fileName, filePath, compilerOpt, config.moduleResolutionHost());
+					const resolved = ts.resolveModuleName(imp.fileName.replace(/\\/g, '/'), filePath, compilerOpt, config.moduleResolutionHost());
```

Comment:
I think `normalizePath` should always replace all backslashes like this here, independent of the host OS - I cannot think of a use case where the host OS should dictate anything, can you?
Maybe the function should just be named `toPosixPath` or something like that.